frontend/packages/console-shared/src/components/cluster-configuration/SaveStatus.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { Alert } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
+
+const SaveStatus: React.FC<{ status?: string }> = ({ status }) => {
+  const { t } = useTranslation();
+  if (status === 'successful') {
+    return (
+      <Alert variant="success" isInline title={t('console-shared~Saved.')}>
+        {t(
+          'console-shared~This config update requires a console rollout, this can take up to a minute and require a browser refresh.',
+        )}
+      </Alert>
+    );
+  }
+  if (status === 'error') {
+    return (
+      <Alert variant="danger" isInline title={t('console-shared~Could not save configuration.')} />
+    );
+  }
```

Comment:
should we show an error message as well here?

Reply:
Added it, looks like this now:

![error](https://user-images.githubusercontent.com/139310/196055672-ffd06597-214f-4840-b101-b271d9577200.png)
![saved](https://user-images.githubusercontent.com/139310/196055676-90cf932d-ba4b-48d8-8184-b40a9047482d.png)